### PR TITLE
npm-load-remove: checkPad.js example

### PR DESCRIPTION
**Pull request by JohnMcLear**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: bin/checkPad.js [hunk 1]
```diff
@@ -5,29 +5,26 @@
...
 // unhandled rejection into an uncaught exception, which does cause Node.js to exit.
-process.on('unhandledRejection', (err) => { throw err; });
 
-const npm = require('ep_etherpad-lite/node_modules/npm');
-const util = require('util');
+const process = require('process');
+const settings = require('ep_etherpad-lite/node/utils/Settings');
+settings.loglevel = 'debug';
 
+process.on('unhandledRejection', (err) => { throw err; });
```
Comment (rhansen) [inline bin/checkPad.js:13]: Move this back to line 8, above all of the `require()` statements (so that an unhandled Promise rejection in one of the includes will cause node to exit).

File: bin/checkPad.js [hunk 2]
```diff
@@ -5,29 +5,26 @@
 
 // As of v14, Node.js does not exit when there is an unhandled Promise rejection. Convert an
 // unhandled rejection into an uncaught exception, which does cause Node.js to exit.
-process.on('unhandledRejection', (err) => { throw err; });
 
-const npm = require('ep_etherpad-lite/node_modules/npm');
-const util = require('util');
+const process = require('process');
```
Comment (rhansen) [inline bin/checkPad.js:9]: You don't need to `require('process')`—the `process` object is a global variable.

File: bin/checkPad.js [hunk 3]
```diff
@@ -5,29 +5,26 @@
...
+const settings = require('ep_etherpad-lite/node/utils/Settings');
+settings.loglevel = 'debug';
```
Comment (rhansen) [inline bin/checkPad.js:11]: Why set the log level?

File: bin/checkPad.js [hunk 4]
```diff
@@ -82,6 +79,6 @@ let checkRevisionCount = 0;
         continue;
       }
     }
-    console.log(`Finished: Checked ${checkRevisionCount} revisions`);
+    console.warn(`Finished: Checked ${checkRevisionCount} revisions`);
```
Comment (rhansen) [inline bin/checkPad.js:82]: I would leave this at `log`.

**Review by rhansen — APPROVED** [review]

File: bin/migrateDirtyDBtoRealDB.js [hunk 5]
```diff
@@ -4,13 +4,8 @@
...
 process.on('unhandledRejection', (err) => { throw err; });
 
-const npm = require('ep_etherpad-lite/node_modules/npm');
-const util = require('util');
-
 (async () => {
-  await util.promisify(npm.load)({});
-
-  process.chdir(`${npm.root}/..`);
+  // process.chdir(`${npm.root}/..`);
```
Comment (rhansen) [inline bin/migrateDirtyDBtoRealDB.js:8]: nuke this line

File: bin/migrateDirtyDBtoRealDB.js [hunk 6]
```diff
@@ -57,6 +52,6 @@ const util = require('util');
   await Promise.all(p);
   console.log(`Wrote all ${numWritten} records`);
 
-  await util.promisify(db.close.bind(db))();
+  await db.close();
```
Comment (rhansen) [inline bin/migrateDirtyDBtoRealDB.js:55]: Undo this change—`db.close()` does not return a Promise.

**Review by rhansen — CHANGES_REQUESTED** [review]

File: bin/migrateDirtyDBtoRealDB.js [hunk 7]
```diff
@@ -36,7 +31,7 @@ const util = require('util');
   await db.init();
 
   console.log('Waiting for dirtyDB to parse its file.');
-  const dirty = dirtyDb('var/dirty.db');
+  const dirty = dirtyDb(`${__dirname}/../var/dirty.db`);
```
Comment (rhansen) [inline bin/migrateDirtyDBtoRealDB.js:34]: This is going to have to change with PR #4707.

**Review by rhansen — CHANGES_REQUESTED** [review]

File: bin/migrateDirtyDBtoRealDB.js (hunk 7 above)
Comment (JohnMcLear) [reply bin/migrateDirtyDBtoRealDB.js:34]: good point, easily done then tho :)

File: bin/migrateDirtyDBtoRealDB.js [hunk 8]
```diff
@@ -52,6 +50,6 @@ process.on('unhandledRejection', (err) => { throw err; });
   await Promise.all(p);
   console.log(`Wrote all ${numWritten} records`);
 
-  await db.close();
+  await util.promisify(db.close.bind(db))();
```
Comment (rhansen) [inline bin/migrateDirtyDBtoRealDB.js]: Do you have to restore the `const util = require('util');` line?

**Review by rhansen — APPROVED** [review]

File: bin/migrateDirtyDBtoRealDB.js (hunk 8 above)
Comment (JohnMcLear) [reply bin/migrateDirtyDBtoRealDB.js]: done
